### PR TITLE
Surface stderr message if available.

### PR DESCRIPTION
**Pull request by rwaldron**
```
$ npm test

> test262-harness@3.3.0 test /Users/rwaldron/clonez/test262-harness
> tap test/*.js

test/test.js ...................................... 114/114
test/validator.js ................................... 20/20
total ............................................. 134/134

  134 passing (24s)

  ok
```

@leobalter @spectranaut this addresses the issue we've run into twice today

### CODE REVIEW

File: lib/validator.js [hunk 1]
```diff
@@ -24,9 +25,10 @@ module.exports = function validate(test) {
         };
       }
     } else if (!ranToFinish && !test.attrs.flags.raw) {
+      let message = result.stderr ? result.stderr : 'Test did not run to completion';
       return {
         pass: false,
-        message: `Test did not run to completion`,
+        message,
```
Comment (leobalter) [inline lib/validator.js]: \o/

File: test/validator.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+  contents: '$DONE();',
+  attrs: {
+    esid: 'sec-foo',
+    description: 'Foo. (Strict Mode)',
+    info: 'Foo',
+    includes: ['assert.js'],
+    flags: {}
+  },
+  async: false,
+  copyright: '// Copyright (C) 2017 Some Person. All rights reserved.\n// This code is governed by the BSD license found in the LICENSE file.\n',
```
Comment (leobalter) [inline test/validator.js]: do we need a sign-off by Some Person?

Comment (rwaldron) [reply test/validator.js]: lol

**Comment by rwaldron** [comment]
Thanks for the review!

**Pull request merged**